frontend/src/components/features/ZakenApiFeatureConfiguration.tsx

```diff
@@ -191,6 +192,24 @@ const ZakenApiFeatureConfiguration = ({
                                                    id="secret"
                                                />
                                            </FormField>
+                                           <FormField
+                                               label={
+                                                   <FormLabel htmlFor={"zaak-types-ids-excluded"}>
+                                                       <FormattedMessage
+                                                           id={"features.zakenapi.zaak-types-ids-excluded"}/>
+                                                   </FormLabel>
+                                               }
+                                               description={
+                                                   <Paragraph>
+                                                       <FormattedMessage
+                                                           id={"features.zakenapi.zaak-types-ids-excluded.description"}/>
+                                                   </Paragraph>
+                                               }>
+                                               <TextInput
```

Comment:
this one is tricky, while technically a text field works bcause we can denote string arrays in spring as a comma separated list, it's not exactly obvious and doesn't match the contract defined above (line 22). I guess this is fine for now for the MVP. Later we do want to have a daragrid-like component for array value, which nlds doesn't currently provision...